scripts/docker_build_all_tagged_commits.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+TAGS=$(git tag)
+DOCKERFILE='Dockerfile'
+DOCKER_REPO='nhsuk/connecting-to-services'
+PUSH_TO_DOCKER='true'
```

Comment:
I thought in the first instance we should keep the script simple, and these variables won't change in the first instance. We can easily add 'getopts' later on: https://rsalveti.wordpress.com/2007/04/03/bash-parsing-arguments-with-getopts/